src/main/java/org/terasology/structureTemplates/internal/systems/ProtectedRegionClientSystem.java

```diff
@@ -58,7 +58,8 @@ private boolean isInProtectedRegion(Collection<Vector3i> positions) {
 
     @ReceiveEvent(priority = EventPriority.PRIORITY_CRITICAL, components = {NoInteractionWhenProtected.class})
     public void onActivationPredicted(ActivationPredicted event, EntityRef target) {
-        Vector3f position = event.getTarget().getComponent(LocationComponent.class).getWorldPosition();
+        Vector3f position = event.getTarget().getComponent(LocationComponent.class).getWorldPosition(new Vector3f());
+        //TODO: is this equivalent to  new Vector3i(position, RoundingMode.HALF_UP);
         Vector3i roundedPosition = new Vector3i(Math.round(position.x), Math.round(position.y), Math.round(position.z));
```

Comment:
there are some changes for rounding that would be relevant. 
https://github.com/JOML-CI/JOML/blob/main/src/org/joml/Vector3i.java#L137-L141